.github/workflows/build_and_run_selected_test.yml

```diff
@@ -0,0 +1,133 @@
+name: Build and Run Selected Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      test-selection:
+        description: 'Specification of selected test for pytest (e.g. regress/test_setup.py)'
+        required: true
+        type: string
+      run-count:
+        description: 'Number of test runs to perform'
+        required: true
+        type: number
+  pull_request:
```

Comment:
A reminder to not to forget to delete `pull_request` trigger before the merge

Reply:
@alexanderlaw I believe this one is the last one thing to do before we can merge it